app/components/Root.jsx

```diff
@@ -1,115 +1,26 @@
 import React, { Component } from 'react';
-
-export default class WinterJokes extends Component {
-  constructor() {
-    super()
-    this.nextJoke = this.nextJoke.bind(this)
-    this.answer = this.answer.bind(this)
-  }
-
-  componentDidMount() {
-    this.nextJoke()
-  }
-
-  nextJoke() {
-    this.setState({
-      joke: randomJoke(),
-      answered: false,
-    })
-  }
-
-  answer() {
-    this.setState({answered: true})
-  }
-
-  render() {
-    if (!this.state) { return null }
-
-    const {joke, answered} = this.state    
+import {BrowserRouter as Router, Route} from 'react-router-dom';
+import Nav from './Nav';
+import Home from './Home';
+import Campuses from './Campus/Campuses';
+import AddCampus from './Campus/AddCampus';
+import AddStudent from './Student/AddStudent';
+import Students from './Student/Students';
+import SingleStudent from './Student/SingleStudent'
+
+export default function Root(props) {
+  //  BELOW ARE ALL THE ROUTES, UPON HITTING A ROUTE THE LISTED COMPONENT WILL RENDER
     return (
-      <div>
-        <h1 onClick={answered ? this.nextJoke : this.answer}>{joke.q}</h1>
-        {answered && <h2>{joke.a}</h2>}
-      </div>
+      <Router>
+        <div className="container">
+          <Route path="/" component={Nav} />
+          <Route exact path="/" component={Home} />
+          <Route exact path="/Campuses" component={Campuses} />
+          <Route exact path="/Campuses/AddCampus" component={AddCampus} />
+          <Route exact path="/Students/AddStudent" component={AddStudent} />
+          <Route exact path="/Students/:StudentName" component={SingleStudent} />
```

Comment:
I think the convention for these prop keys would be`:studentName` (starting with a lower case letter like `let studentName` or `{ studentName: "" }`).